src/api/store/message.py

```diff
@@ -383,16 +369,23 @@ def send_message(self, context, args) -> Tuple[dict, MassEnergizeAPIError]:
             audience = args.get("audience", None)
             schedule = get_schedule(args.get("schedule", None))
             communities = args.get("community_ids", None)
-            recipients = get_message_recipients(audience, audience_type, sub_audience_type, communities)
-            
 
+            print("=== communities ===", communities)
+            recipients = get_message_recipients(audience, audience_type)
+
+            logo = ME_LOGO_PNG
+            
+            associated_community = None
             if not recipients:
                 return None, InvalidResourceError()
             
             user = get_user_from_context(context)
             if not user:
                 return None, InvalidResourceError()
             
+            if not context.user_is_super_admin:
+                associated_community = Community.objects.filter(id=communities[0]).first()
```

Comment:
question - is there a case where multiple communities will be passed, perhaps for a community admin of multiple communities?  In that case, picking the first one won't always be correct.   (fix later)